.circleci/create_circleci_config.py

```diff
@@ -363,8 +363,8 @@ def create_circleci_config(folder=None):
             all_test_list = f.read()
     else:
         all_test_list = []
-    if len(all_test_list) > 0:
-        jobs.extend(PIPELINE_TESTS)
+
+    jobs.extend(PIPELINE_TESTS)
```

Comment:
I will remove this part before merge. It's used to make sure the TF pipeline tests all pass with vision installed.